ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -1,47 +1,34 @@
 from styx_msgs.msg import TrafficLight
-import cv2
+from keras.models import model_from_json
+from scipy import misc
 import numpy as np
+import rospkg
+import tensorflow as tf
 
 class TLClassifier(object):
     def __init__(self):
         #TODO load classifier
-        pass
-
-    def simple_opencv_red_color_classifier(self,image):
```

Comment:
Do not remove OpenCV-based classifier. Inherit from TLClassifier and do the same for Pradeep's OpenCV-based classifier.